packages/esbuild-why/index.js

```diff
@@ -1,14 +1,16 @@
-let { visualizer } = require('esbuild-visualizer')
-let { join } = require('path')
-let { writeFileSync } = require('fs')
+const { visualizer } = require('esbuild-visualizer')
```

Comment:
I use `let` :)

Reply:
Oh, also at the top-level?

I looked up the `eslint-plugin-prefer-let` it says you can use `const` at top-level. 😛 

I can change that.

Reply:
I use `const` only for cases like `const MAX = 100`